docs/designers-developers/developers/themes/theme-support.md

```diff
@@ -134,6 +134,46 @@ Themes are responsible for creating the classes that apply the colors in differe
 
 The class name is built appending 'has-', followed by the class name _using_ kebab case and ending with the context name.
 
+### Block Gradient Presents
+
+Different blocks have the possibility of selecting from a list of predined of gradients. The block editor provides a default gradient presets, but a theme can overwrite them and provide its own:
+
+```php
+add_theme_support(
+	'__experimental-editor-gradient-presets',
```

Comment:
Any particular reason to keep this as experimental for now? When do you think we should make it stable?